client.go

```diff
@@ -107,11 +108,12 @@ func (c *Client) ProvisionResource(ctx context.Context, cbID manifold.ID,
 	model ResourceBody) (string, bool, error) {
 
 	body := models.ResourceRequest{
-		ID:       model.ID,
-		Product:  manifold.Label(model.Product),
-		Plan:     manifold.Label(model.Plan),
-		Region:   models.RegionSlug(model.Region),
-		Features: model.Features,
+		ID:         model.ID,
+		Product:    manifold.Label(model.Product),
+		Plan:       manifold.Label(model.Plan),
+		Region:     models.RegionSlug(model.Region),
+		Features:   model.Features,
+		PlatformID: model.PlatformID,
```

Comment:
Like I mentioned in the other PR maybe we should switch to domain, but will leave it up to you
![image](https://user-images.githubusercontent.com/1552848/63095088-b101ad00-bf40-11e9-913b-a3436cce9dae.png)